spring-cloud-data-flow/spring-cloud-data-flow-stream-processing/data-flow-server/pom.xml

```diff
@@ -42,8 +53,9 @@
     </dependencyManagement>
 
     <properties>
-        <spring-cloud-dataflow-dependencies.version>1.1.0.RELEASE</spring-cloud-dataflow-dependencies.version>
-        <spring-cloud-dependencies.version>Brixton.SR7</spring-cloud-dependencies.version>
+        <spring-cloud-dataflow-dependencies.version>1.3.1.RELEASE</spring-cloud-dataflow-dependencies.version>
+        <spring-cloud-dependencies.version>Edgware.SR6</spring-cloud-dependencies.version>
+        <hibernate.compatible.version>5.2.12.Final</hibernate.compatible.version>
```

Comment:
We are having a number of compatibility issues, similar to these ones:

https://spring.io/blog/2017/11/29/spring-cloud-data-flow-1-3-0-m3-released
https://github.com/spring-cloud/spring-cloud-dataflow/issues/334
https://stackoverflow.com/questions/48915459/build-error-when-upgrade-spring-cloud-starter-dataflow-server-local-to-1-3-0